src/widgets/switch/lv_switch.h

```diff
@@ -28,6 +28,16 @@ extern "C" {
 
 LV_ATTRIBUTE_EXTERN_DATA extern const lv_obj_class_t lv_switch_class;
 
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef enum {
```

Comment:
It is recommended to use the existing enumeration `lv_dir_t`
`LV_DIR_NONE` as auto mode.